test/indexer.py

```diff
@@ -0,0 +1,42 @@
+"""For convenience, load test data into Elasticsearch index "test".
+
+Unfortunately there's no easy way to index a JSON file using curl. (See
+https://stackoverflow.com/questions/23798433/json-bulk-import-to-elasticstearch.)
+So write a Python script.
+"""
+
+import csv
+import time
+
+from elasticsearch import Elasticsearch
+from elasticsearch import helpers
+
+
+INDEX_NAME = 'test'
+
+
+def init_elasticsearch():
+  # Wait for Elasticsearch to come up.
+  time.sleep(10)
```

Comment:
Added comment. The right way is to use wait_for_status, but I [had trouble getting that to work](https://discuss.elastic.co/t/cant-get-python-client-wait-for-status-to-work/123163). I think sleep is good enough for now. FYI the right number is somewhere between 5 and 10 seconds, so we're not losing that many seconds.